ca_cdk_constructs/edge_services/waf_v2_builder.py

```diff
@@ -53,10 +53,10 @@ def __init__(
         scope: Construct,
         name: str,
         description: str,
-        tags: dict = dict(),
-        waf_scope: str = "CLOUDFRONT",
-        log_group: cf_logs.LogGroup = None,
-        default_action: waf.CfnWebACL.DefaultActionProperty = None,
+        tags: Optional[dict] = dict(),
```

Comment:
not related to the PR but I don't think the construct should be dealing with tags

Reply:
You are probably right